docker-entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+# Add elasticsearch as command if needed
+if [[ "$1" == "-"* ]]; then
```

Comment:
I think this could be rendered slightly more explicit (from the mongo entrypoint).
```bash
if [ "${1:0:1}" = '-' ]; then
```